### ISSUE TITLE
Too Many Errors

### ISSUE THREAD

**Issue by notexploit1337** [Dionis1902/AccountGeneratorHelper#11]
This repository has too many errors, on gmailnator too much errors, other mail services have errors too please fix as soon as possible.

**Comment by jodiekurnia** [comment]
this is opensource code, come on give the creator some appreciation ...

**Comment by Dionis1902** [comment]
Could you specify exactly which errors?

**Comment by notexploit1337** [comment]
![Uploading Screenshot_2023-03-28-11-06-35-03_820b80da67ca34a12f493dfc8307f092.jpg…]()

**Comment by notexploit1337** [comment]
![Screenshot_2023-03-28-11-06-35-03_820b80da67ca34a12f493dfc8307f092](https://user-images.githubusercontent.com/121854648/228138570-01993f1b-f5da-4b8e-8730-2c77d6c4ad74.jpg)

**Comment by notexploit1337** [comment]
```py
from account_generator_helper import GmailNator

mail = GmailNator()
email = mail.get_email_online()
print(email)
while True:
  inb = mail.get_inbox()
  print(inb)
  input("refresh")
```
My code

**Comment by notexploit1337** [comment]
@jodiekurnia ye i appreciate owner, but these errors.......

**Comment by Dionis1902** [comment]
Fixed in [Version 1.0.14](https://github.com/Dionis1902/AccountGeneratorHelper/releases/tag/1.0.14)
